homeassistant/components/media_player/sonos.py

```diff
@@ -1164,3 +1187,9 @@ def update_alarm(self, **data):
     def device_state_attributes(self):
         """Return device specific state attributes."""
         return {ATTR_IS_COORDINATOR: self.is_coordinator}
+
+    def get_play_mode(self):
+        """Return play mode (only shuffle and normal is supported atm)."""
+        if self._shuffle is True:
```

Comment:
**Preference:**
Similar to above, this can just be:
```python
return 'SHUFFLE' if self._shuffle else 'NORMAL'
```

Reply:
This is now addressed. Thanks for the suggestion